syft/formats/common/spdxhelpers/license_test.go

```diff
@@ -12,74 +12,125 @@ func Test_License(t *testing.T) {
 	tests := []struct {
 		name     string
 		input    pkg.Package
-		expected string
+		expected struct {
```

Comment:
doens't need to change in this PR, but in the future an in-function struct could be declared since it's used a lot:
```
func MyTest(t *testing.T) {
  type expected struct {
    concluded string
    declared string
   }

  tests := []struct{...}
  ...
}
```